src/AccessibilityInsights.SharedUx/Controls/SettingsTabs/ApplicationSettingsControl.xaml

```diff
@@ -180,7 +180,7 @@
                     <Label Name="lblDelay" Margin="0,5,0,0" Content="{x:Static Properties:Resources.lblDelayContent}" Padding="0" Style="{StaticResource LblText}" Target="{Binding ElementName=tbMouseDelay}"/>
                     <TextBox Name="tbMouseDelay" Height="26" Width="38" Margin="5,0,5,0" PreviewTextInput="textboxMouseDelay_PreviewTextInput" MaxLines="1" MaxLength="4" 
                      VerticalContentAlignment="Center" HorizontalContentAlignment="Right" Style="{StaticResource TxtBxText}" AutomationProperties.Name="{x:Static Properties:Resources.tbMouseDelayAutomationName}" TextChanged="tbMouseDelay_TextChanged"/>
-                    <Label Content="ms" Margin="0,5,0,0" Padding="0" Style="{StaticResource LblText}"/>
+                    <Label Content="milliseconds" Margin="0,5,0,0" Padding="0" Style="{StaticResource LblText}"/>
```

Comment:
If you wanted to, this string should probably be pulled out to the resources file. Approving PR since this isn't a regression

Reply:
Agreed that this is a separate PR. I'm pretty sure we have multiple non-resource strings, as well as multiple duplicate strings